pypesto/objective/amici/amici.py

```diff
@@ -458,18 +439,23 @@ def call_unprocessed(
 
         x_dct = self.par_arr_to_dct(x)
 
-        # only ask amici to compute required quantities
         amici_reporting = (
             self.amici_reporting
             if amici_reporting is None
             else amici_reporting
         )
         if amici_reporting is None:
```

Comment:
It should be made clear in the documentation that amici_reporting takes precedence over return_dict imo.